DOCS-1112: Clarify RPi raspi-config (#2142)
viamrobotics/docs: docs/installation/prepare/rpi-setup.md

## Enable Communication Protocols

> Certain hardware, such as analog-to-digital converters (ADCs), accelerometers, and sensors, communicate with your Pi using specialized communications protocols, including I2C, SPI, serial, or one-wire protocols.

Reviewer: [nit] "Certain hardware" is singular but then the list is of plural items so "communicate" works for the plural but not the singular.

## Enable Communication Protocols

> If you are using hardware that requires I2C, SPI, serial, or one-wire protocols to communicate with your Pi, you will need to enable them using `raspi-config`.
> These protocols are required to support certain hardware, such as analog-to-digital converters (ADCs), accelerometers, and sensors.

Reviewer: [opt] To me, the current way sounds almost redundant/odd because it's essentially saying twice that it's required. I think I like the second suggestion  best because it keeps the lists in separate sentences whilst introducing the why before the how, though it makes "them" ambiguous.
```suggestion
Certain pieces of hardware, such as analog-to-digital converters (ADCs), accelerometers, and sensors, communicate with your Pi using I2C, SPI, serial, or one-wire protocols.
If you are using hardware that requires any of these protocols, you need to enable them using `raspi-config`:
```
```suggestion
Certain hardware requires I2C, SPI, serial, or one-wire protocols to communicate with your Pi.
If you are using hardware such as an analog-to-digital converter (ADC), accelerometer, or sensor that requires any of these protocols, you need to enable them using `raspi-config`:
```

Reviewer 2: I slightly like the first way better

1. Launch the config tool by running the following command:

   ```sh {class="command-line" data-prompt="$"}
   sudo raspi-config
   ```

1. Use your keyboard to select "Interface Options", and press return.

   {{< imgproc alt="Screenshot of the Raspi Config screen with a red box and red arrow pointing to the '3 Interface Options' option where you can find the I2C and other drivers" src="/installation/rpi-setup/Installation-Raspberry-Pi-I2C-Raspi-Config-Interfacing-Options.png" resize="800x" declaredimensions=true >}}

1. Enable the relevant protocols to support your specific hardware:

   - If you are using an analog-to-digital converter (ADC), enable **SPI**.

>    - If you are using an accelerometer, enable **I2C**.

Reviewer 2: I feel this is too specific, LOTS of hardware use i2c.  I think we need to either give specific examples (and say "for example") or just tell them to check their hardware specs

   - If you are using a CSI v1.3 or v2.0 camera, enable **Legacy Camera** support.

>    - If you are using a movement sensor or power sensor, you will likely need to enable the **Serial Port**.

Reviewer 2: Actually some use i2c.  And that's kind of the point, it really varies by hardware.  Some even have both i2c and spi interfaces.
